### ISSUE TITLE
at-function-named-arguments should ignore maps

### ISSUE THREAD

**Issue by RCopeland** [kristerkari/stylelint-scss#236]
I have a map like: 
```
$space-values: (
  tiny: $horizontal-list-spacing-tiny,
  small: $horizontal-list-spacing-small,
  medium: $horizontal-list-spacing-medium,
  large: $horizontal-list-spacing-large,
  huge: $horizontal-list-spacing-huge
);
```
which is triggering scss/at-function-named-arguments. 

I'm using stylelint@8.40 and stylelint-scss@2.2.0.

**Comment by RCopeland** [comment]
```
$space-values: (
  tiny: $horizontal-list-spacing-tiny,
  small: $horizontal-list-spacing-small,
  medium: $horizontal-list-spacing-medium,
  large: $horizontal-list-spacing-large,
  huge: $horizontal-list-spacing-huge
) !default;
```
makes the error go away...

**Comment by kristerkari** [comment]
I created a fix:
#237

**Comment by kristerkari** [comment]
Fix released in v3.0.1

**Comment by RCopeland** [comment]
great thanks!
